src/sentry/api/bases/organization_events.py

```diff
@@ -300,7 +300,7 @@ def handle_results_with_meta(
             fields_meta = meta.get("fields", {})
 
             if standard_meta:
-                isMetricsData = meta.pop("isMetricsData", False)
+                isMetricsData = meta.get("isMetricsData", False)
```

Comment:
Had to do this since `pop` was removing `isMetricsData` for subsequent series in top n metas